EIPS/eip-eof-meta.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
```suggestion
---
eip: 7692
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.